### PR TITLE
Adding usages field to new-ontology-request.md

### PR DESCRIPTION
**Pull request by matentzn**
This makes it possible to actually pass the dashboard entirely

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/new-ontology-request.md [hunk 1]
```diff
@@ -73,4 +73,7 @@ tracker: https://github.com/YOURORG/your_repo/issues
 license:
   url: http://creativecommons.org/licenses/by/3.0/
   label: CC-BY
+usages:
```
Comment (nlharris) [inline .github/ISSUE_TEMPLATE/new-ontology-request.md:76]: "usages" sounds weird. could you say "examples of use" or does the keyword there have to be one word? if it has to be one word, maybe "uses"? or just "usage"

File: .github/ISSUE_TEMPLATE/new-ontology-request.md [hunk 2]
```diff
@@ -73,4 +73,7 @@ tracker: https://github.com/YOURORG/your_repo/issues
...
+  - user: http://website.of.the.project.using.my.ontology.com
+    description: A short description how the project above is using my ontology
```
Comment (nlharris) [inline .github/ISSUE_TEMPLATE/new-ontology-request.md:78]: > description: A short description how the project above is using my ontology

description: A short description of how the project above is using my ontology

**Review by nlharris — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/new-ontology-request.md (hunk 1 above)
Comment (matentzn) [reply .github/ISSUE_TEMPLATE/new-ontology-request.md:76]: this is not up to me - this is all official obo metadata! its all decided centrally there!

File: .github/ISSUE_TEMPLATE/new-ontology-request.md (hunk 2 above)
Comment (matentzn) [reply .github/ISSUE_TEMPLATE/new-ontology-request.md]: thx

File: .github/ISSUE_TEMPLATE/new-ontology-request.md (hunk 1 above)
Comment (nlharris) [reply .github/ISSUE_TEMPLATE/new-ontology-request.md:76]: ah, ok, if "usages" is the standard label here, then so be it.

**Review by nlharris — APPROVED** [review]

**Pull request merged**